src/layman/util.py

```diff
@@ -291,6 +291,18 @@ def get_workspace_publication_url(publication_type, workspace, publication_name,
     return publ_module.get_workspace_publication_url(workspace, publication_name, x_forwarded_items=x_forwarded_items)
 
 
+def get_publication_url(publication_type, uuid, use_cache=True, *, x_forwarded_items=None):
+    if publication_type == 'layman.map':
```

Comment:
I suggest to define `layman.map` as package constant, so it will be easier to find all occurences once we would like to remove such hacks.